src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/Body.js

```diff
@@ -1,11 +1,10 @@
 // @flow
-import type {ChildrenArray, Element} from 'react';
+import type {ChildrenArray} from 'react';
 import React from 'react';
-import Row from './Row';
 import type {ButtonConfig, SelectMode} from './types';
 
 type Props = {
-    children?: ChildrenArray<Element<typeof Row>>,
+    children?: ChildrenArray<*>,
```

Comment:
Doesn't the `Body` only allow `Row`s as children? Then this would be wrong...

Reply:
Everyone of the mentioned components except the table is throwing flow errors when the children type is set. I guess it's because those components are inside React.Children.map/forEach. `ChildrenArray<*>` is a bit more specific than `any`.

Reply:
You know that I've fixed that without `*` or `any` in the latest commit in this PR?